### ISSUE TITLE
Accessibility 2022

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/almanac.httparchive.org#2889]
# Accessibility 2022

![Accessibility illustration](https://raw.githubusercontent.com/HTTPArchive/almanac.httparchive.org/main/src/static/images/2019/accessibility/hero_lg.jpg)

If you're interested in contributing to the Accessibility chapter of the 2022 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide).

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| ---- | ------- | --------- | -------- | ------- | ----------- |
|  | @SaptakS @scottdavis99 @thibaudcolas | @shantsis | @thibaudcolas | @dereknahman | @foxdavidj |

<details>
<summary><b>Expand for more information about each role 👀</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [**0. Form the content team**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
+ [x] **May 1**: The content team has at least one author, reviewer, and analyst

### [**1. Plan content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
+ [x] **May 15** The content team has completed the chapter outline in the [draft doc][~google-doc]

### [**2. Gather data**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
+ [x] **June 1**: Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/custom-metrics/blob/main/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress
- **June 1 - 15**: HTTP Archive runs the June crawl

### [**3. Validate results**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
+ [x] **August 1**: Analysts have queried all metrics and saved the output to the [results sheet][~google-sheets]

### [**4. Draft content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
+ [x] **September 1**: The content team has written, reviewed, and edited the chapter in the [doc][~google-doc]

### [**5. Publication**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
+ [x] **September 15**: The completed chapter and all required [metadata](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters) and [figures](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide) are converted to markdown and submitted to [GitHub][~chapter-markdown]
- **September 26**: Target launch date 🚀

## Chapter resources

Refer to these 2022 Accessibility resources throughout the content creation process:

[📄 Google Docs][~google-doc] for outlining and drafting content
[🔍 SQL files][~sql-files] for committing the queries used during analysis
[📊 Google Sheets][~google-sheets] for saving the results of queries
[📝 Markdown file][~chapter-markdown] for publishing content and managing public metadata
[💬 #web-almanac-accessibility on Slack][~slack-channel] for team coordination

[~google-doc]: https://docs.google.com/document/d/1Q6PEEfqD59vLgrOXY2VQUDeVrl0wcWHqsQiIhQ0Bt3M/edit?usp=sharing
[~sql-files]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2022/accessibility
[~google-sheets]: https://docs.google.com/spreadsheets/d/1ladaKh6RbtMKQwkccwxDJGQf85KyhfLrtlM_9e9sLH8/edit?usp=sharing
[~chapter-markdown]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en/2022/accessibility.md
[~slack-channel]: https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA

**Comment by ericwbailey** [comment]
We chatted about this a little bit last year, but I was wondering about people's thoughts on scattering accessibility across applicable chapters this year?

The reasoning is:

- Accessibility is holistic, and touches on many chapters' concerns
- Previous accessibility chapters are long, which disincentives reading in full
- People who aren't into accessibility may not read the accessibility chapter, which limits its exposure

**Comment by tunetheweb** [comment]
It is mentioned in a few chapters as can we seen in this search: https://almanac.httparchive.org/en/search?q=accessibility

Mobile Web, in particular usually gives it some coverage, as ensuring content is accessibly even on smaller devices is important, but also the Media, CMS and Jamstack chapters have touched upon it - often the Lighthouse scores provide a nice, high-level measure for those wanting to quickly summarise a technology’s Accessibility (and yes I know it’s limited in what it can measure!).

When we edit, and another chapters topic is mentioned, we always to try ensure likes back to the full chapter on the topic are included to encourage people to the chapter if they want to learn more about it.

Open to other ideas you have on how we can do more though?

**Comment by shantsis** [comment]
I've seen it linked across multiple chapters while editing last year (I recommended a few of those connections :) ), but they don't go anywhere near the depth the accessibility chapter does.

Also I'm open to reviewing again!

**Comment by ericwbailey** [comment]
>Open to other ideas you have on how we can do more though?

For the MDN, we adopted [a specific subsection for it](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#accessibility_concerns). The idea is it'd meet the developer where they were, but also had enough space to go into detail if needed.

**Comment by alextait1** [comment]
I really like the idea of it being scattered across the chapters as well as its own dedicated chapter. I think it's helpful to have a place to send people and to show it as a priority through both having its own chapter and being woven throughout the almanac. Kinda similar to my view that we need accessibility specialist and everyone has to have involvement and knowledge.

Also hi! I'd like to be a reviewer this year to make room for new writers, a new lead and honestly because my workload got a little bananas last year balancing writing/chapter lead stuff with work and family life. 

I will be happy to support whoever wants to take on the chapter lead role this year 🎉

**Comment by rviscomi** [comment]
I agree with @alextait1, I think this topic deserves a dedicated chapter while still being woven into the related chapters.

**Comment by alextait1** [comment]
I'd really love to further explore what integrating into other chapters could look like too @ericwbailey

**Comment by SaptakS** [comment]
I'd love to co-author.

**Comment by scottdavis99** [comment]
I'd love to co-author the Accessibility chapter again this year!

**Comment by scottdavis99** [comment]
One thing I've been doing in my a11y advisory work is taking a "Top 5" approach to fixing Accessibility issues, based on the WebAIM Million Top 5. There aren't any surprises on the list: Color Contrast, Img Alt Text, Input Labels, Link text, and Buttons. By focusing on the most common errors, we've been able to see dramatic measurable improvements. 

Perhaps this year, the chapter could take an "actionable" approach -- biggest impact issues mentioned first, with suggestions for corrections -- rather than the "informational" approach from previous years.

**Comment by rviscomi** [comment]
> Perhaps this year, the chapter could take an "actionable" approach -- biggest impact issues mentioned first, with suggestions for corrections -- rather than the "informational" approach from previous years.

I would love to see that!

**Comment by ericwbailey** [comment]
>Perhaps this year, the chapter could take an "actionable" approach -- biggest impact issues mentioned first, with suggestions for corrections -- rather than the "informational" approach from previous years.

I love this idea.

**Comment by foxdavidj** [comment]
@scottdavis99 @scottdavis99 @SaptakS @shantsis @alextait1

Hey everyone, excited to see we've already got a full team (aside from an analyst) for the chapter!

To kick things off, I'd love to set up a 30 minute call within the next two weeks to put any new faces to names, and start the planning and brainstorming process. 

@scottdavis99 as the Chapter Lead can you assist in finding a time that works for everyone? You can see my availability via my calendly here: https://calendly.com/foxdavid/30min

Also, here is an agenda for what we might want to discuss on the kickoff call: https://docs.google.com/document/d/1nqPjvY0zg2YJda3ghIJuvVaSeY_wbz_nwBK15tHULsg/edit?usp=sharing

**Comment by dereknahman** [comment]
Hi everyone. If you still have space for an Editor, I'd love to put myself forward.

**Comment by scottdavis99** [comment]
Hi Kirsten,

We’d love to have you onboard!

I’m on vacation this week, but I’ll try to wrangle time on everyone’s calendar to meet up this week or next. 

Cheers,
S

**Comment by foxdavidj** [comment]
@scottdavis99 Sounds good. Just a heads up that the Chapter Outline is due by May 15th, so having the kickoff meeting before then would be ideal so there is time left to work on the outline

**Comment by scottdavis99** [comment]
Good catch! Thx!
-s

**Comment by thibaudcolas** [comment]
Hi all, I’d love to help with this 🙂  It looks like analysts are what’s needed the most right now – this isn’t a role I have experience with myself, but I think I can pick it up if needed. I’m also interested in being a reviewer if we got other volunteer analysts.

**Comment by scottdavis99** [comment]
Hi Thibaud,

We’d love to have you help with this chapter. I am the lead author, and I was planning to be the analyst for this chapter as well, but there’s always room for more interested folks like you. 

I’m out on holiday for the rest of this week, but we’ll try to schedule our first coordination meeting next week. 

Cheers,
S

**Comment by alextait1** [comment]
For what it's worth @scottdavis99 having been an author and then the chapter lead - it's A LOT of work to do well, I would recommend taking all the help you can get! If I'd needed to do @foxdavidj's analyst work last year I would have had a really difficult time since leading requires a lot of admin and unofficially being a reviewer too. Just wanted to chime in to help save you some pain lol :)

**Comment by rviscomi** [comment]
@scottdavis99 @SaptakS @shantsis @alextait1 @ericwbailey @thibaudcolas @dereknahman make sure you all have access to the [planning doc](https://docs.google.com/document/d/1Q6PEEfqD59vLgrOXY2VQUDeVrl0wcWHqsQiIhQ0Bt3M/edit?usp=sharing) and start adding your ideas to the outline. What are some new topics in accessibility this year, or what would be good to revisit from previous years? To reiterate what @foxdavidj said earlier, the outline is due on May 15, which is set up to give us enough time to make any instrumentation changes to the June crawl before it kicks off. I know the team is still getting settled and scheduling your first sync, but it'd be good for everyone to start brainstorming on the outline to stay on track. Thanks!

**Comment by foxdavidj** [comment]
@scottdavis99 How is the outline for the chapter coming along? I'm not seeing much of one in the planning doc. Have you been working with the team in another Google doc?

We're passed the May 15th deadline, and quickly heading running out of time to write any new custom metrics for the chapter (1 weeks left to have them written, approved, and merged).

**Comment by foxdavidj** [comment]
@scottdavis99 Also, how big of an ask would it be for me to request, once a week on Friday, an update on how the chapter is doing and any progress/hurdles/questions/etc you have. And post it to the github chapter issue? Can be a simple bullet list.

It’d help me a lot with staying in the loop (and not nagging you) and making sure I’m helping you solve problems you’re facing.

Thoughts?

**Comment by foxdavidj** [comment]
@scottdavis99 Making the weekly updates here would be great. Can you take a look at the Chapter Google Doc? It looks like @thibaudcolas left a couple questions in there for you

**Comment by foxdavidj** [comment]
@scottdavis99 How is the outline coming along? Looks like there is still work to be done and we're nearly a month past the deadline.

@scottdavis99 @thibaudcolas Now that the crawl has started, please create a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/2937)) to track the progress of writing the queries needed for the chapter

**Comment by foxdavidj** [comment]
@scottdavis99 friendly ping

**Comment by foxdavidj** [comment]
Hey @scottdavis99, we're fast approaching the deadline for having all of the analysis completed (Aug 1). But it doesn't look like we have a chapter outline, or any queries planned out (yet alone written). Could you give us an update on when you'd be able to get that completed?

⚠️ Given that this chapter is behind on outlining and querying data for the Aug 1st deadline, it's at risk of getting cut from the Almanac unless we can make progress on it soon. To get it back on track we'd need a chapter outline and draft PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/2937)) completed ASAP

**Comment by rviscomi** [comment]
⛔ @scottdavis99 I see that you've been active in the planning doc recently, which is great, but we're getting a bit concerned as we haven't gotten a status update from you a while.

The chapter still seems to be blocked on Milestone 1, completing the outline, which was due in May. We're looking for something resembling a table of contents to sketch out the topics you'd like to explore this year. The chapter is also missing Milestone 2, a draft of the analysis PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/2959)), which was due in June. And as @foxdavidj mentioned, Milestone 3, completing the analysis, is due next week.

Given how far behind this chapter is, it's at very high risk of falling irrecoverably behind, which puts us in the unfortunate position of having to decide whether to cut it from this year's publication. **If you can check in with us and confirm that you'll be able to catch up to Milestone 2 by the end of this week, I think we'll be able to get it back on track.**

Do you need any additional help with the analysis? Once we see what metrics need to be queried, I or someone else can help with writing them. Alternatively, we can bootstrap the chapter with all of last year's queries to get you up and running sooner.

I know this year has been tough for many, especially with Covid, so we will completely understand if you don't feel up to it any more and need to bow out. Let us know if that's what you need to do—no hard feelings, I promise.

Wish you the best and hope to hear from you soon.

**Comment by rviscomi** [comment]
@scottdavis99 @SaptakS @shantsis @ericwbailey @thibaudcolas @dereknahman @foxdavidj

Sorry everyone, but given that we still don't have the list of metrics that need to be queried we have to make the tough call to close the chapter.

Thank you all for signing up and we hope that you consider contributing next year. If this frees you up to help other chapters and you're still interested, take a look at the [list of open chapters](https://github.com/HTTPArchive/almanac.httparchive.org/labels/2022%20chapter) and leave a comment to jump in.

**Comment by thibaudcolas** [comment]
Sob. I had completely forgotten signing up to this in-between holidays, and only just realised with Scott DM-ing me on Slack. I’m sorry not to have made more of an effort on this earlier 😞 

From my early research on the custom metrics already in place, queries written in past years and discussions on Slack, it seemed there were plenty enough metrics already set up for interesting accessibility analysis – so as far as I could tell, we  were done with one half of milestone 2, with the draft PR missing. Does this make any difference to your decision? It might have been a mistake on my part to spend so much time on custom metrics when everything we had considered adding to the chapter didn’t need any (in particular media queries and anything CSS-related).

---

> Alternatively, we can bootstrap the chapter with all of last year's queries to get you up and running sooner.

If we also reused the outline, all of the charts, and as much of the editorial content as possible – seems to me like we could create a chapter with commensurably much less effort. Would that still be an option? Run a copy of 2021’s outstanding chapter, with up-to-date data? Though we had other ambitions initially, I’m sure this would nonetheless be valuable to a lot of people – and we could catch up on milestones 1, 2, 3 in a matter of days, with a much simpler milestone 4.

**Comment by rviscomi** [comment]
I've had some discussions with @scottdavis99 @SaptakS and @thibaudcolas these past couple of days and want to give an update for all contributors.

The chapter is still closed. It's fallen too far behind for us to plan on including it in this year's publication. BUT the people I've chatted with are still very interested to salvage the chapter.

The plan is for @thibaudcolas to rerun all of the 2021 queries against the 2022 dataset and @SaptakS will take the lead on writing the chapter with help from @scottdavis99 if interested.

If the chapter can meet the following requirements, we'll be able to reverse the decision to close it:

- The chapter must be data-driven, so there needs to be a substantial amount of data backing it up. Reusing the 2021 outline and queries would accomplish this.
- The content must be original, not a simple rewording of last year's authors.
- The chapter must be peer reviewed and edited by someone other than the authors, following the guidelines in the wiki ([Reviewers' Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [Editors' Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)).
- The chapter must be fully caught up with all milestones by the publication deadline on September 15, no exceptions.

We'll continue to provide BigQuery access so that analysts are not querying out of pocket, but as part of closing the chapter, we won't be providing project coordination support with regular progress check-ins.

Anyone who still wants to contribute to the chapter is welcome to do so. Please reply here and/or coordinate in the Slack channel about tackling the remaining work.

To be clear, I am strongly discouraging this plan because of the enormous amount of work to be done in a short amount of time. But I won't stop anyone from trying to get it back on track, and I'll commit to reversing the decision if the requirements are met. Thanks everyone for your understanding, this is a tough decision and I'd like to be as fair as possible.

**Comment by shantsis** [comment]
I am still open to reviewing the chapter if we go down this route

**Comment by ericwbailey** [comment]
I'm over-committed and the deadline crunch would risk my level and quality of involvement. Best of luck, and looking forward to next year.

**Comment by alextait1** [comment]
> 

Same goes for me. I hope to be involved and see this chapter better supported next year, it really matters.

**Comment by thibaudcolas** [comment]
For anyone still following this – here’s a quick progress update:

- I’ve created the copy of the 2021 queries for 2022: #3057. I believe this is ready for review by HTTP Archive analysts. This wasn’t just copy-pasting, I made sure to review the whole of the chapter to identify exactly which queries are used where, and made according fixes along the way (#3039, #3040, #3043, #3047, #3048, #3049, #3063).
- I ran the resulting queries and validated their results [side-by-side with those of 2021](https://docs.google.com/spreadsheets/d/1eJtp09ja2FLoY4OTcDR3LdunGDsE62Gm3ZCcRF_7r8c/edit#gid=744885125). This includes generating the corresponding figures, so will save quite a bit of time once we reach the next phase.

The above means we’ve reached [milestone 2. Gather data](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data), and takes us quite close to reaching [milestone 3. Validate results](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results). To formally catch up with that milestone, we still need:

- For the analysts to approve / merge #3057.
- For me to add the results over to the [2022 results sheet](https://docs.google.com/spreadsheets/d/1ladaKh6RbtMKQwkccwxDJGQf85KyhfLrtlM_9e9sLH8/edit?usp=sharing). I quickly tried this with a few queries to make sure I had a good idea of the process – but will wait for the queries to be reviewed / approved before spending more time on this.

And that should be it!

---

I’m hoping the above will take up to a week and we’ll then have a week to catch up with the chapter lifecycle in time for [milestone 4. Draft content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content). @SaptakS had made progress on this already but I’ll let him share his own update as he sees fit.

From my side, I’ll generate the Markdown for all of the figures and update the captions.

---

I’ve also taken a few notes on additional reporting we could do without writing more queries. The simplest ones seem to be:

1. Accessibility-related media queries – `(hover: hover)`, `(forced-colors: active)`, `pointer: coarse`, etc.
2. Year-over-year 2021-2022 comparisons. 

Both types of data only require generating new figures. I’d be interested in working on those but only if we managed to catch up with milestone 4 on time.

**Comment by tunetheweb** [comment]
PR looks mostly good to me so feel free to start running the queries.

As I noted on the PR, I think it would be good to also include Desktop Lighthouse data (new this year!) to give a new angle on what you can include in the report. However I wouldn’t let that hold you up running the other queries (or maybe even also the LH queries with just mobile initially) to unblock the chapter and can do that after (either in that PR or as a follow up).

**Comment by thibaudcolas** [comment]
Hi everyone, the [Accessibility (Web Almanac 2022) sheet](https://docs.google.com/spreadsheets/d/1ladaKh6RbtMKQwkccwxDJGQf85KyhfLrtlM_9e9sLH8/edit) now contains all of the results, pivot tables, and figures based on our queries.

I’ve validated this by comparing the results with the [2021 sheet](https://docs.google.com/spreadsheets/d/1WjAM5ZnHjMQt-rKyHvj2eVhU_WdzzFTjpoYWMr_I0Cw/edit), since a lot of the queries are identical. In particular, I made a [2021 - 2022 sheet](https://docs.google.com/spreadsheets/d/1eJtp09ja2FLoY4OTcDR3LdunGDsE62Gm3ZCcRF_7r8c/edit) that has results for both years if anyone wants to compare them.

Back to the 2022 sheet – it follows the same formatting as 2021’s where possible, with the same sheet titles where possible. For some queries, I’ve also left the data from 2021 in there, and occasionally made additional YoY comparisons, in the cases where it seemed most interesting. There’s no plan to use those currently though, we’ll get the outline as agreed finished first.

## Finishing milestone 3

@SaptakS I believe you’ve already reviewed some or all of the results? Let me know if you want clarifications on any aspect of the data. @shantsis as a reviewer, we could also use your help if you have the time. [Milestone 3](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results) specifically says:

> Authors and reviewers should review the results for comprehension and correctness. If the data for a metric is not clear, they should ask for clarification. If the data looks wrong or unintuitive, they should point that out so the analyst could modify/rerun the query if necessary or explain why the results are correct.

## [Milestone 4. Draft content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)

Just focusing on my remit:

- [x] The figure visuals are already created for all results, following the styleguide
- [ ] I have yet to write accessible descriptions for all the figures
- [ ] I’ll then generate images for all figures so I can add them to the draft content
- [ ] And then generate the Markdown for all figures (I have a script that can do this semi-automatically)

I’ll try to finish all of this in my evening today so @SaptakS has 10 days to go through other parts of this milestone

## [Milestone 5. Publication](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)

@dereknahman you had volunteered to help as an editor, is this still something you’re interested in? If not, we don’t have anyone else lined up so we’ll need to figure something out.

**Comment by SaptakS** [comment]
@thibaudcolas That's right. I have reviewed some of them. I have finished writing a draft version based on the analysis, so I would say we are mostly on time. It would be great if you could help me in copy-pasting or importing the graphs to the docs draft.

**Comment by dereknahman** [comment]
Hey @thibaudcolas. I'm happy to edit when you need me - just say when!

**Comment by thibaudcolas** [comment]
Took me a bit longer than I was expecting but I’ve finally finished adding all of the query results in the draft chapter content. There are 21 graphs, 8 "big numbers", 2 tables, and 9 sets of results mentioned within the copy.

For each, I’ve added the embed from Google Sheets where appropriate, the Markdown `{{ figure_markup() }}` or `{{ figure_link() }}` including all up-to-date metadata, and a comment (`//`) for review purposes only with a link to the sheet, and to the SQL query.

I added all of this where Saptak had earmarked placement of figures for current draft content, and added everything else following the draft outline afterwards.

---

I don’t think there’s much else for me to do at least as an analyst for the time being, unless we were to have enough time for additional results. For now I might spend some time with a reviewer hat and look at content drafted to date.

@shantsis I believe the first half of the chapter’s content should now be fully ready to review. For the second half, the figures’ visuals, captions, and alternative text should be ready to review.

**Comment by thibaudcolas** [comment]
@dereknahman 👋 it looks like Saptak has drafted [all of the chapter](https://docs.google.com/document/d/1Q6PEEfqD59vLgrOXY2VQUDeVrl0wcWHqsQiIhQ0Bt3M/edit#heading=h.6zeuu0wsj6l9) aside from the last section. I think this would be a good time for you to get involved!

**Comment by dereknahman** [comment]
Hi! I’ll start looking at it this evening. As is inevitable, this coming week features home renovations, a separate project deadline and a full-on work week. I’ll only be able to work on this in the evenings. I’ll aim to bat it back to you by this weekend latest

**Comment by thibaudcolas** [comment]
Hi all, just a quick update for people who keeping up with Slack conversations that we now have the whole chapter content written, reviewed, and edited. What’s left to do is two corrections + Markdown conversion:

1. [ ] One figure needing correction + updating its description accordingly (with me)
2. [ ] One sentence to potentially remove (not sure who is making that call)
3. [ ] Markdown conversion + pull request + generation of images (me converting, Saptak reviewing)

Once that’s done – there are two potential additions we are considering, and one section I would like to further rework if there is time. This would all be based on existing data so it seems doable, though a stretch.

There is one other potential addition based on existing data which I made a figure for but we’ll likely leave out, and 5 additions requiring new queries which I had a go at but we’ll leave out too (I’ll share them nonetheless in due time for future reference).

---

We’ll have caught up on Milestone 4 once I’ve made those two corrections later today. And assuming the Markdown conversion goes to plan, we should be caught up with Milestone 5 in one or two days, before the deadline.

@rviscomi once we’ve done both of those things, do you have any thoughts on whether we can / should consider the additional changes mentioned above? Are we allowed to make further tweaks hypothetically all the way until September 26th, or is there a cut-off date after which we should consider the content frozen?

**Comment by thibaudcolas** [comment]
FYI the Markdown conversion and image generation is done: #3087

We now need further review of the Markdown to make sure the conversion didn’t create issues, and there is further metadata missing from the chapter (featured quotes and stats)

**Comment by rviscomi** [comment]
Congrats everyone on getting the chapter back on track, it's great to see it nearing the finish line! 🎉

> @rviscomi once we’ve done both of those things, do you have any thoughts on whether we can / should consider the additional changes mentioned above? Are we allowed to make further tweaks hypothetically all the way until September 26th, or is there a cut-off date after which we should consider the content frozen?

Content should be locked down at this point, to prevent any publishing delays. If you really need to make changes, do so before the markdown is reviewed/submitted.
